doc/lsst.pipe.base/creating-a-pipelinetask.rst

```diff
@@ -411,14 +434,16 @@ take into account that a background may or may not be supplied.
 
     ...
 
-    class ApertureTask(pipeBase.PipelineTask):
 
+    class ApertureTask(pipeBase.PipelineTask):
         ...
 
-        def run(self, exposure: afwImage.Exposure,
-                inputCatalog: afwTable.SourceCatalog,
-                background: typing.Optional[afwMath.BackgroundList] = None
-                ) -> pipeBase.Struct:
+        def run(
+            self,
+            exposure: afwImage.Exposure,
+            inputCatalog: afwTable.SourceCatalog,
+            background: typing.Optional[afwMath.BackgroundList] = None,
```

Comment:
Should we use more modern `| None` instead?

Reply:
Right. I haven't got a way of running `pyupgrade` on the docs whereas `blacken-docs` is a command I can run. I'll try to clean these up.